### PR TITLE
feat(NewRelic): Add tracking tags

### LINKED ISSUE #2283: NR - Create browser monitor for docs.royalnavy.io
Dependent on #2282

### LINKED ISSUE #2284: NR - Create browser monitor for storybook.royalnavy.io

### PR DESCRIPTION
**Pull request by m7kvqbe1**
## Related issue

Closes #2283 
Closes #2284

## Overview

Add NewRelic tracking tags for Storybook and legacy docs-site.

### CODE REVIEW

**Review by elglup — APPROVED** [review]
LGTM

File: packages/react-component-library/.storybook/preview-head.html [hunk 1]
```diff
@@ -6,3 +6,10 @@
   .sbdocs-h2:first-of-type { margin-bottom: 40px; }
   .rn-sidebar--storybook { position: unset !important; max-height: 30rem; }
 </style>
+
+<script type="text/javascript">
```
Comment (thyhjwb6) [inline packages/react-component-library/.storybook/preview-head.html:11]: I wonder if there's a way to make it more obvious what this is for? Either separating it out or a comment 😄

**Review by thyhjwb6 — APPROVED** [review]
Looks good, minor comment about whether it could be more "self documenting"

File: packages/react-component-library/.storybook/preview-head.html (hunk 1 above)
Comment (m7kvqbe1) [reply packages/react-component-library/.storybook/preview-head.html:11]: Added a HTML comment.

**Pull request merged**